src/graph/graph.c

```diff
@@ -1119,6 +1148,55 @@ void Graph_BulkDelete(Graph *g, Node *nodes, uint node_count, Edge *edges, uint
 	if(edge_deleted != NULL) *edge_deleted = _edge_deleted;
 }
 
+int Graph_UpdateEntity
+(
+	Graph *g,
+	GraphEntity *ge,
+	Attribute_ID attr_id,
+	SIValue new_value,
+	EntityType entity_type
```

Comment:
Please make sure function is well commented, each attribute should be accompanied with a comment